src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchLagPorts.groovy

```diff
@@ -0,0 +1,56 @@
+package org.openkilda.functionaltests.helpers.model
+
+import static org.openkilda.functionaltests.model.cleanup.CleanupActionType.DELETE_LAG_LOGICAL_PORT
+
+import org.openkilda.functionaltests.model.cleanup.CleanupManager
+import org.openkilda.model.SwitchId
+import org.openkilda.northbound.dto.v2.switches.LagPortRequest
+import org.openkilda.northbound.dto.v2.switches.LagPortResponse
+import org.openkilda.testing.service.northbound.NorthboundServiceV2
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+import groovy.util.logging.Slf4j
+
+@Slf4j
+@EqualsAndHashCode(excludes = 'northboundV2, cleanupManager')
+@ToString(includeNames = true, excludes = 'northboundV2, cleanupManager')
+class SwitchLagPorts {
+
+    SwitchId switchId
+
+    @JsonIgnore
+    NorthboundServiceV2 northboundV2
+    @JsonIgnore
+    CleanupManager cleanupManager
+
+    SwitchLagPorts(NorthboundServiceV2 northboundV2, CleanupManager cleanupManager, SwitchId switchId) {
+        this.switchId = switchId
+        this.northboundV2 = northboundV2
+        this.cleanupManager = cleanupManager
+    }
+
+    LagPortResponse createLogicalPort(Set<Integer> portNumbers, boolean lacpReply = null) {
```

Comment:
If you're going to call this method only from `SwitchExtended` class, then you should move this method to the one. 
If you're going to allow users create LAG ports with `new()`, I'd recommend to make this method static and rename to `create()`, to have something like `LagPort.create()` in tests.